specification/communications-manager.yaml

```diff
@@ -49,6 +49,25 @@ paths:
 
         Use this endpoint to send a batch of messages to 1 or more NHS patients.
 
+        ### References
+
+        You must provide two reference values within the payload to this endpoint:
+
+        * A message batch reference
+        * A per message reference
+
+        Both of these references must be UUIDs.
```

Comment:
Given UUID is an acronym, should we use the full term followed by (UUIDs)

Reply:
Good shout, I'll make this part better.

Reply:
I've given the full term, the acronym plus I've linked to wikipedia so the consumer can find more information on UUIDs if they have never used them before.